src/_includes/docs/debug/debug-flow-windows.md

```diff
@@ -0,0 +1,123 @@
+#### Build the Windows version of the Flutter app in PowerShell or the Command Prompt
+
+To generate the needed Windows platform dependencies,
+run the `flutter build` command.
+
+```terminal
+C:\> flutter build windows --debug
+```
+
+```terminal
+Building Windows application...                                    31.4s
+√  Built build\windows\runner\Debug\my_app.exe.
+```
+
+#### Start debugging with Visual Studio first
+
+If you use Visual Studio to debug most of your code, start with this section.
+
+##### Start the local Windows debugger
+
+1. To open the project solution file, go to
+   **File** <span aria-label="and then">></span>
+   **Open** <span aria-label="and then">></span>
+   **Project/Solution…**
+
+   You can also press <kbd>Control</kbd> + <kbd>Shift</kbd> + <kbd>O</kbd>.
+
+1. Choose the `build/windows/my_app.sln` file in your Flutter app directory.
+
+{% comment %}
+   ![Open Project/Solution dialog box in Visual Studio 2022 with my_app.sln file selected.](/assets/images/docs/testing/debugging/native/visual-studio/choose-solution.png){:width="100%"}
+   <div markdown="1">{:.figure-caption}
+   Open Project/Solution dialog box in Visual Studio 2022 with
+   `my_app.sln`` file selected.
+   </div>
+{% endcomment %}
```

Comment:
There's an important step here: you need to tell Visual Studio which project to launch. If you don't do this once, Visual Studio will error saying that it cannot launch the project. Visual Studio will remember which project to launch.

You can do this by right clicking on the `my_app` project and selecting `Set as Startup Project`. For example: 

![Visual Studio project context menu](https://user-images.githubusercontent.com/737941/215009513-d31c59fd-1f54-44d9-a702-6dd3fdf71492.png)